src/test/compile-fail/coerce-overloaded-autoderef.rs

```diff
@@ -17,24 +20,32 @@ fn borrow2<T>(_: &mut T, _: &T) {}
 fn double_mut_borrow<T>(x: &mut Box<T>) {
     let y = borrow_mut(x);
     let z = borrow_mut(x);
-    //~^ ERROR cannot borrow `*x` as mutable more than once at a time
+    //[ast]~^ ERROR cannot borrow `*x` as mutable more than once at a time
+    //[mir]~^^ ERROR cannot borrow `*x` as mutable more than once at a time (Ast)
```

Comment:
the `~^^` here could just be `~|`

Reply:
Cf. https://github.com/rust-lang/rust/pull/44811#discussion_r141350517